eks-worker-al2-variables.json

```diff
@@ -23,7 +23,7 @@
     "pause_container_version": "3.5",
     "pull_cni_from_github": "true",
     "remote_folder": "/tmp",
-    "runc_version": "1.1.5-1.amzn2",
+    "runc_version": "1.1.*",
```

Comment:
What's the version which would get install if we don't add anything here?

Reply:
If we specified `*` here, we would always get the latest. I've scoped it down a bit so we don't install, e.g. `2.0` without warning and break the build.

Reply:
The latest version available is `runc-1.1.7-2.amzn2`, which will be pulled in by `1.1.*` obvs